wordpress-mobile/WordPress-Android
# Gravatar permission-denied analytics

This adds the permission name to the Gravatar related permission-denied events to gather more insight on which permissions the user doesn't feel comfortable granting.

To test:

* On Android v6 and above
* Revoke the app's permissions or uninstall and reinstall the app
* Tap the Gravatar in /me screen
* Deny some or all permissions
* Notice the permission names be tracked (the AndroidStudio Debugger can be used for that)

Also, this PR handles the case where the permission dialog is interrupted (see https://developer.android.com/reference/android/support/v4/app/ActivityCompat.OnRequestPermissionsResultCallback.html) though I couldn't find a way to test it as the dialog does not respond to the `Back` button.
